Makefile

```diff
@@ -460,3 +468,17 @@ ci-env-down:
 	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(ENV_DOCKER_COMPOSE) down
 	@$(call func_echo_success_status, "$@ -> [ Done ]")
+
+
+### eclint: Validate files against editorconfig
```

Comment:
Could you move the dev kit stuff to a separate PR? Thanks!

Reply:
i've dropped the commit